resources/testharness.js

```diff
@@ -2510,6 +2510,7 @@ policies and contribution forms [3].
         this.test_done_callbacks = [];
         this.all_done_callbacks = [];
 
+        this.do_notify_test_state = true;
```

Comment:
Oh sorry I missed this before you added more but it probably makes sense to make this `hide_test_state = false` since we generally want the default value for booleans to be false.